docs/source/en/guides/inference.md

```diff
@@ -127,23 +146,46 @@ code as before, changing only the `model` parameter:
 >>> client.text_to_image(..., model="https://uu149rez6gw9ehej.eu-west-1.aws.endpoints.huggingface.cloud/deepfloyd-if")
 ```
 
+Note that you cannot specify both a URL and a provider - they are mutually exclusive. URLs are used to connect directly to deployed endpoints.
+
 ### Authentication
 
-Calls made with the [`InferenceClient`] can be authenticated using a [User Access Token](https://huggingface.co/docs/hub/security-tokens).
-By default, it will use the token saved on your machine if you are logged in (check out
-[how to authenticate](https://huggingface.co/docs/huggingface_hub/quick-start#authentication)). If you are not logged in, you can pass
-your token as an instance parameter:
+Authentication depends on which provider you are using:
+
+1. For the default Hugging Face Inference API, you can authenticate using a [User Access Token](https://huggingface.co/docs/hub/security-tokens):
 
 ```python
 >>> from huggingface_hub import InferenceClient
 >>> client = InferenceClient(token="hf_***")
 ```
 
+By default, it will use the token saved on your machine if you are logged in (see [how to authenticate](https://huggingface.co/docs/huggingface_hub/quick-start#authentication)).
+
+2. For third-party providers, you have two options:
+
+**Direct access to provider**: Use your own API key to interact directly with the provider's service:
+```python
+>>> client = InferenceClient(
+    provider="replicate",
+    api_key="r8_****"  # Your Replicate API key
+)
+```
+
+**Proxied through Hugging Face** : Use Hugging Face as a proxy to access third-party providers. Simply specify
+your Hugging Face token and the provider you want to use. The calls will be routed through Hugging Face's infrastructure
+using our provider keys, and the usage will be billed directly to your Hugging Face account:
+```python
+>>> client = InferenceClient(
+    provider="replicate",
+    token="hf_****"  # Your HF token 
+)
+```
+
 <Tip>
 
-Authentication is NOT mandatory when using the Inference API. However, authenticated users get a higher free-tier to
-play with the service. Token is also mandatory if you want to run inference on your private models or on private
-endpoints.
+Authentication is NOT mandatory when using the Hugging Face Inference API. However, authenticated users get a higher 
+free-tier to play with the service. Token is also mandatory if you want to run inference on your private models or 
+on private endpoints.
 
 </Tip>
```

Comment:
I would remove this Tip altogether (quite outdated). I think InferenceAPI requires authentication now (or very very limited free-tier) and running InferenceAPI on private models is disabled.